backtest.py

```diff
@@ -59,14 +49,17 @@ def process_file_pair(v1_file, v2_file, v1_dir, v2_dir, all_pools_info):
     results['chain'] = all_pools_info['chain']
     results['symbol'] = all_pools_info['symbol']
 
-    differences = []
+    abs_differences = []
+    rel_differences = []
     for pool in all_pools_info.index:
```

Comment:
It's not apparent here what we are taking the differences of.  It's also not apparent in the sheet.  Can we add this?  I think right now it's a sum of a bunch of values on each row, but maybe I'm reading wrong.